src/main/docker/run-pct.sh

```diff
@@ -146,12 +146,7 @@ mkdir -p "${PCT_OUTPUT_DIR}"
 # Determine if we test the plugin against another JDK
 ###
 TEST_JDK_HOME=${TEST_JAVA_ARGS:-"/usr/lib/jvm/java-${JDK_VERSION:-8}-openjdk-amd64"}
-if [[ "${JDK_VERSION}" = "11" ]] ; then
-  echo "JDK_VERSION detected is 11. Adding JAXB and modules to the command lines."
```

Comment:
Wouldn't it be interesting to print the version used (just the version, not the modules)? or perhaps it is printed in any other place?

Reply:
PCT report does it